enterprise/frontend/src/embedding-sdk/cli/snippets/express-server-snippet.ts

```diff
@@ -0,0 +1,132 @@
+import { HARDCODED_USERS } from "embedding-sdk/cli/constants/hardcoded-users";
+
+import { HARDCODED_JWT_SHARED_SECRET } from "../constants/config";
+
+interface Options {
+  instanceUrl: string;
+}
+
+const DEFAULT_EXPRESS_SERVER_PORT = 4477;
+
+// TODO: inject customerId into the users
```

Comment:
TODO: the dashboard currently errors with "query requires user attribute `customer_id`", because of the sandboxing. We need to inject the user's sampled customer_id from their database into the user attributes.